[2022-05-06 11:22:16,872] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 manual__2022-05-06T05:52:14.709011+00:00 [queued]>
[2022-05-06 11:22:16,876] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 manual__2022-05-06T05:52:14.709011+00:00 [queued]>
[2022-05-06 11:22:16,876] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-05-06 11:22:16,876] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-05-06 11:22:16,876] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-05-06 11:22:16,883] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_1> on 2022-05-06 05:52:14.709011+00:00
[2022-05-06 11:22:16,884] {standard_task_runner.py:52} INFO - Started process 1465703 to run task
[2022-05-06 11:22:16,885] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_1', 'manual__2022-05-06T05:52:14.709011+00:00', '--job-id', '43', '--raw', '--subdir', '/home/sanjeev/Documents/SubDAGs/dags/paralle_processing.py', '--cfg-path', '/tmp/tmpajhiim5n', '--error-file', '/tmp/tmp_pm_5zvk']
[2022-05-06 11:22:16,886] {standard_task_runner.py:77} INFO - Job 43: Subtask task_1
[2022-05-06 11:22:16,912] {logging_mixin.py:109} INFO - Running <TaskInstance: parallel_dag.task_1 manual__2022-05-06T05:52:14.709011+00:00 [running]> on host kumar
[2022-05-06 11:22:16,943] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_1
AIRFLOW_CTX_EXECUTION_DATE=2022-05-06T05:52:14.709011+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-06T05:52:14.709011+00:00
[2022-05-06 11:22:16,943] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-06 11:22:16,943] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 4']
[2022-05-06 11:22:16,947] {subprocess.py:85} INFO - Output:
[2022-05-06 11:22:20,949] {subprocess.py:93} INFO - Command exited with return code 0
[2022-05-06 11:22:20,986] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_1, execution_date=20220506T055214, start_date=20220506T055216, end_date=20220506T055220
[2022-05-06 11:22:21,033] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-06 11:22:21,060] {local_task_job.py:264} INFO - 2 downstream tasks scheduled from follow-on schedule check
